frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -22,6 +23,27 @@ const applyPods = (podsData: PodRingData, dc: PodRCData) => {
   return podsData;
 };
 
+export const podRingLabel = (resourceObj: K8sResourceKind, isScalingAllowed: boolean) => {
+  const title =
+    resourceObj.status.availableReplicas || (isScalingAllowed ? 'Scaled to 0' : 'Autoscaled');
+
+  const subTitle =
+    resourceObj.spec.replicas !== resourceObj.status.availableReplicas
+      ? !resourceObj.spec.replicas
+        ? isScalingAllowed
+          ? ''
+          : 'to 0'
+        : `scaling to ${resourceObj.spec.replicas}`
+      : resourceObj.spec.replicas > 1 || resourceObj.spec.replicas === 0
```

Comment:
We have a 'pluralize' util function that can be used here.

https://github.com/openshift/console/blob/96ef6ea8615e8f4ec17a784feb6bfd69a369da25/frontend/public/components/utils/details-page.tsx#L18

Reply:
I was aware of this function. But here we have a different use case this function return number with pluralize string. But we need only pluralize string.